anta/tests/hardware.py

```diff
@@ -269,3 +269,53 @@ def test(self) -> None:
         total_adverse_drop = command_output.get("totalAdverseDrops", "")
         if total_adverse_drop != 0:
             self.result.is_failure(f"Incorrect total adverse drops counter - Expected: 0 Actual: {total_adverse_drop}")
+
+
+class VerifyRedundancyProto(AntaTest):
+    """Verifies that the redundancy protocol status.
+
+    Expected Results
+    ----------------
+    * Success: The test will pass if the expected redundancy protocol is configured and operational, and if switchover is ready.
+    * Failure: The test will fail if the expected redundancy protocol is not configured, not operational, or if switchover is not ready.
+    * Skipped: The test will be skipped if the peer supervisor card is not inserted.
+
+    Examples
+    --------
+    ```yaml
+    anta.tests.hardware:
+      - VerifyRedundancyProto:
+    ```
+    """
+
+    categories: ClassVar[list[str]] = ["hardware"]
+    commands: ClassVar[list[AntaCommand | AntaTemplate]] = [AntaCommand(command="show redundancy status", revision=1)]
+
+    class Input(AntaTest.Input):
+        """Input model for the VerifyRedundancyProto test."""
+
+        redundency_proto: Literal["unknownProtocol", "sso", "rpr", "simplex"] = "sso"
+        """Configured redundancy protocol."""
+
+    @skip_on_platforms(["cEOSLab", "vEOS-lab", "cEOSCloudLab", "vEOS"])
+    @AntaTest.anta_test
+    def test(self) -> None:
+        """Main test function for VerifyRedundancyProto."""
+        self.result.is_success()
+        command_output = self.instance_commands[0].json_output
+
+        # Verify peer supervisor card insertion
+        if command_output["peerState"] == "notInserted":
+            self.result.is_skipped("Peer supervisor card not inserted")
+
+        # Verify that the expected redundancy protocol is configured.
+        elif command_output["configuredProtocol"] != self.inputs.redundency_proto:
+            self.result.is_failure(f"Redundancy protocol {self.inputs.redundency_proto} not configured")
+
+        # Verify that the expected redundancy protocol configured and operational
```

Comment:
If you use `is`, please use it everywhere for consistency. Same thing for periods `.`.
```suggestion
        # Verify that the expected redundancy protocol is configured and operational.
```

Reply:
Thanks for the suggestion, Carl. You're right, other test cases don't use "is" or end with a period in comments, so we're removing them here too.